.github/ISSUE_TEMPLATE.md

```diff
@@ -1,28 +1,42 @@
+**PLEASE READ:** Before reporting here, *please* carefully read our **Troubleshooting Guide** at <https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ> and search the issues page here for your error/problem, as most postd bugs are duplicates or easy fixes.
+
+If you do post an issue here, please describe step by step in detail what you were doing when the error occurred,. Otherwise, your issue will likely be closed after 7 days. Thanks.
+
+**Note:** Delete this message before submitting.
```

Comment:
Then we could remove this one, which is a bit misleading.

Reply:
See above.